dashboard/config/scripts/levels/Dance_Party_01.level

```diff
@@ -39,14 +39,14 @@
     "custom_helper_library": "World.SPRITE_NAMES = [\"CAT\", \"MOOSE\"];\r\n\r\nWorld.MOVE_NAMES = [\r\n  {\r\n    name: \"Rest\",\r\n    mirror: true\r\n  },\r\n  {\r\n    name: \"ClapHigh\",\r\n    mirror: true\r\n  },\r\n  {\r\n    name: \"Clown\",\r\n    mirror: false\r\n  },\r\n  {\r\n    name: \"Dab\",\r\n    mirror: true\r\n  },\r\n  {\r\n    name: \"DoubleJam\",\r\n    mirror: false\r\n  },\r\n  {\r\n    name: \"Drop\",\r\n    mirror: true\r\n  },\r\n  {\r\n    name: \"Floss\",\r\n    mirror: true\r\n  }\r\n];",
     "preload_asset_list": [
       "https://curriculum.code.org/images/DancePartyLoading.gif",
-      "https://curriculum.code.org/images/sprites/dance_20181127/cat_08.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/cat_09.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/cat_10.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/cat_11.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/moose_24.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/moose_25.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/moose_26.png",
-      "https://curriculum.code.org/images/sprites/dance_20181127/moose_27.png"
+      "https://curriculum.code.org/images/sprites/dance_grouped/cat_08.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/cat_09.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/cat_10.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/cat_11.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/moose_24.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/moose_25.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/moose_26.png",
+      "https://curriculum.code.org/images/sprites/dance_grouped/moose_27.png"
```

Comment:
@joshlory @ryansloan This looks wrong.  `dance_20181127` is the latest and greatest.